src/server/enterprise/server/api_server.go

```diff
@@ -240,6 +240,14 @@ func (a *apiServer) Activate(ctx context.Context, req *ec.ActivateRequest) (resp
 	}); err != nil {
 		return nil, err
 	}
+
+	// Wait until watcher observes the write
+	for {
+		time.Sleep(time.Second) // give other pachd nodes time to observe the write
+		if t := a.enterpriseExpiration.Load().(time.Time); !t.IsZero() {
+			break
+		}
+	}
```

Comment:
Another place where I think it'd be good to use our backoff library and have a notification when we retry so we have a chance of catching the bug if this becomes an infinite loop.

Reply:
Same as above. This now uses backoff + sleep.